console/src/app/modules/policies/oidc-webkeys/oidc-webkeys-table/oidc-webkeys-table.component.html

```diff
@@ -0,0 +1,57 @@
+<cnsl-refresh-table (refreshed)="refresh.emit()" [loading]="(dataSource$ | async) === null">
```

Comment:
We implicitly expect this to be in a loading state, but if an error occurs, it will continuously spin. I'd either provide a loading state or introduce another state for the check to distinguish between loading and error states.